core/src/validator.rs

```diff
@@ -601,15 +601,15 @@ impl Validator {
             &config.snapshot_config.bank_snapshots_dir,
         );
 
-        info!("Cleaning orphaned account snapshot directories..");
+        info!("Cleaning orphaned account snapshot directories...");
```

Comment:
If you want to move this one to three periods, want to adjust the one above in `Cleaning accounts paths` as well for the sake of consistency